tests/test_units.py

```diff
@@ -97,4 +157,4 @@ def test_minmax_operationtime():
     mm.outputs["energy"][:] = 400
     mm.outputs["energy"][-1:] = 0
     runtime = mm.get_operation_time(datetime(2023, 7, 2))
-    assert runtime == 5
+    assert runtime == 4
```

Comment:
We set all outputs to 400 and then the last one to 0, but since the last one is at 2023-07-02 00:00:00, all hours before the time we check for are positive. Therefore, get_operation_time() returns the minimum operation time of 4.